spec/take_away_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'take_away'
+ 
+  describe TakeAway do
+
+  	
+
+  	it "customer can enter restaurant" do
+  		expect(TakeAway).to respond_to(:new)
+  	end
+
+  	it "can view menu prices" do
+  		expect(subject).to respond_to(:show_menu)		
+  	end
+
```

Comment:
This is okay but you could also test what the response of the method should be. So your method should actually return a string (again, you probably do not want to return the actual menu object) and then your test should check that the return value is equal to that string.